pom.xml

```diff
@@ -641,7 +641,7 @@
           <version>1.12.0</version>
           <configuration>
             <installDirectory>${project.build.directory}</installDirectory>
-            <nodeVersion>v12.16.2</nodeVersion>
```

Comment:
How are we testing this? Was it tested locally?

Reply:
Yea - tested locally.